client/src/components/standing_table/standingData.ts

```diff
@@ -0,0 +1,61 @@
+/* Home page Data */
+export const homeStandingHeader = ['Pos', 'Club', 'Pl', 'GD', 'Pts'];
+
+export interface HomeStanding {
+  name: JSX.Element;
+  played: number;
+  GD: number;
+  points: number;
+}
+
+export const homeScorerHeader = ['Pos', 'Player', 'Club', 'Goals'];
```

Comment:
Maybe I put it like this, but please update the naming to correct format; `HomeScorerHeader`.

Don't ever assume everything I put in is correct

Reply:
Fixed!